Bundled.Dockerfile

```diff
@@ -0,0 +1,31 @@
+# This Dockerfile is used to create a build of lsif-clang
+# along with the accompanying dynamically linked libraries.
+# Those are packaged up together in a tarball for ease
+# of deployment. In the future, it would be nicer to provide
+# a statically linked binary.
+#
+# See issue https://github.com/sourcegraph/lsif-clang/issues/72
+
+FROM ubuntu:22.04 as build
```

Comment:
See my note, building on such a new version of Ubuntu greatly restricts the platforms that can run the resulting release.